src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -40,11 +40,17 @@ public function process(ContainerBuilder $container)
             $container->removeDefinition('twig.extension.yaml');
         }
 
+        $viewDir = \dirname(new \ReflectionClass('Symfony\Bridge\Twig\Extension\FormExtension'), 2).'/Resources/views';
+
+        if ($container->has('mailer')) {
+            $emailPath = $viewDir.'/Email';
```

Comment:
This new path should also be added to the `twig.template_iterator` service to make it cacheable on `cache:clear`.

Reply:
done